scripts/ocsf-schemas.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python3
+
+from pathlib import Path
+import sys
+import textwrap
+import typing
+
+import requests
+
+# ============= Configuration ============= #
+URL = "https://schema.ocsf.io/export/schema"
+DOCUMENT_ENTITIES = True
+DOCUMENT_FIELDS = False
+CLASS_PREFIX = "ocsf."
+OBJECT_PREFIX = "ocsf.object."
+COLUMN_LIMIT = 80
+# ========================================= #
+
+OMIT_MARKER = object()
+BASIC_TYPES = {
+    "boolean_t": "bool",
+    "float_t": "double",
+    "integer_t": "int64",
+    # TODO: Don't omit this.
+    "json_t": OMIT_MARKER,
+    "long_t": "int64",
+    "string_t": "string",
+    "bytestring_t": "blob",
+    "datetime_t": "time",
+    "ip_t": "ip",
+    "subnet_t": "subnet",
+    # TODO: Is this the best choice?
+    "timestamp_t": "time",
+}
+
+# TODO: The schema defines some recursive types which cannot be represented
+# faithfully. Hence, we omit some fields to break recursive cycles. Instead of
+# that, we could recursively unfold the types up to a certain depth, or perhaps
+# use a slightly different representation.
```

Comment:
I think discarding these is good enough for a proof of concept, but we probably want to instantiate these fields to a depth of 1 or 2 for actual use.

Reply:
Agreed, but let's do that in a follow-up.